funding/commitment_type_negotiation.go

```diff
@@ -269,6 +269,25 @@ func explicitNegotiateCommitmentType(channelType lnwire.ChannelType, local,
 
 		return lnwallet.CommitmentTypeSimpleTaproot, nil
 
+	// Simple taproot channels with scid and zero conf.
+	case channelFeatures.OnlyContains(
+		lnwire.SimpleTaprootChannelsRequired,
+		lnwire.ZeroConfRequired,
+		lnwire.ScidAliasRequired,
+	):
+
+		if !hasFeatures(
+			local, remote,
+			lnwire.SimpleTaprootChannelsOptional,
+			lnwire.ZeroConfOptional,
+			lnwire.ScidAliasRequired,
```

Comment:
should remove `ScidAliasRequired` in the call to `hasFeatures` -- see this case as an example https://github.com/lightningnetwork/lnd/blob/4a03074fe80e5ac4f4516de1857fcfd9cfb011ea/funding/commitment_type_negotiation.go#L111-L122